ui/components/component-library/text/text.types.ts

```diff
@@ -88,7 +90,7 @@ export interface TextProps extends BoxProps {
    * The text-align of the Text component. Should use the TEXT_ALIGN object from
    * ./ui/helpers/constants/design-system.js
    */
-  textAlign?: keyof typeof TEXT_ALIGN;
+  textAlign?: TextAlign;
   /**
```

Comment:
Adding so TEXT_ALIGN object works. This will eventually be replaced when https://github.com/MetaMask/metamask-extension/issues/18444 is completed